app/Services/Model/Imp/SummitService.php

```diff
@@ -1501,6 +1501,19 @@ public function addSummit(array $data)
                 }
             }
 
+            $registration_slug_prefix = $data['registration_slug_prefix'] ?? null;
```

Comment:
this is not correct
bc the SLUG get transformed when its set
so u need to externalize this

 $slug_prefix = strtoupper(trim(str_replace(" ","_", $registration_slug_prefix)));
on a static function and pass the user data by this function before doing the search

Reply:
done